tests/e2e/support/objects/app-files/spaces/actions.ts

```diff
@@ -310,3 +313,14 @@ export const removeExpirationDateFromMember = async (args: {
   await openSharingPanel(page)
   await Collaborator.removeExpirationDateFromCollaborator({ page, collaborator })
 }
+
+export const downloadSpace = async (page: Page): Promise<void> => {
+  await openActionsPanel(page)
+  const [download] = await Promise.all([
+    page.waitForEvent('download'),
+    page.locator(downloadSpaceSelector).click()
+  ])
+  await sidebar.close({ page })
+
+  expect(download.suggestedFilename()).toContain('download.tar')
```

Comment:
can we return this value here and assert it in the step definition instead?
```suggestion
  return download
```